src/main/java/fi/otavanopisto/kuntaapi/server/integrations/gtfs/GtfsTranslator.java

```diff
@@ -0,0 +1,173 @@
+    result.setId(kuntaApiId.getId());
+    result.setName(serviceCalendar.getServiceId().getId());
+    result.setDays(parseScheduleDays(serviceCalendar));
+    result.setStartDate(parseServiceDateTime(serviceCalendar.getStartDate()));
+    result.setEndDate(parseServiceDateTime(serviceCalendar.getEndDate()));
+    result.setExceptions(parseScheduleExceptions(exceptions));
+    
+    return result;
+    
+  }
+  
+  public Route translateRoute(PublicTransportRouteId kuntaApiId, org.onebusaway.gtfs.model.Route gtfsRoute, PublicTransportAgencyId kuntaApiAgencyId) {
+    Route result = new Route();
+    result.setId(kuntaApiId.getId());
+    result.setAgencyId(kuntaApiAgencyId.getId());
+    result.setShortName(gtfsRoute.getShortName());
+    result.setLongName(gtfsRoute.getLongName());
+    
+    return result;
+  }
+
+  public Stop translateStop(PublicTransportStopId kuntaApiId, org.onebusaway.gtfs.model.Stop gtfsStop) {
+    Stop result = new Stop();
+    result.setId(kuntaApiId.getId());
+    result.setName(gtfsStop.getName());
+    result.setLat(gtfsStop.getLat());
+    result.setLng(gtfsStop.getLon());
+    
+    return result;
+  }
+
+  public StopTime translateStopTime(PublicTransportStopTimeId kuntaApiId, org.onebusaway.gtfs.model.StopTime gtfsStopTime, PublicTransportStopId kuntaApiStopId, PublicTransportTripId kuntaApiTripId) {
+    StopTime result = new StopTime();
+    result.setId(kuntaApiId.getId());
+    result.setDepartureTime(gtfsStopTime.getDepartureTime());
+    result.setArrivalTime(gtfsStopTime.getArrivalTime());
+    result.setStopId(kuntaApiStopId.getId());
+    result.setSequency(gtfsStopTime.getStopSequence());
+    result.setDistanceTraveled(gtfsStopTime.getShapeDistTraveled());
+    result.setTripId(kuntaApiTripId.getId());
+    
+    return result;
+  }
+  
+  public Trip translateTrip(PublicTransportTripId kuntaApiId, org.onebusaway.gtfs.model.Trip gtfsTrip, PublicTransportRouteId kuntaApiRouteId, PublicTransportScheduleId kuntaApiScheduleId) {
+    Trip result = new Trip();
+    result.setId(kuntaApiId.getId());
+    result.setRouteId(kuntaApiRouteId.getId());
+    result.setScheduleId(kuntaApiScheduleId.getId());
+    result.setHeadsign(gtfsTrip.getTripHeadsign());
+    
+    return result;
+  }
+  
+  private List<ScheduleException> parseScheduleExceptions(List<ServiceCalendarDate> exceptions) {
+    List<ScheduleException> results = new ArrayList<>(exceptions.size());
+    for(ServiceCalendarDate exception : exceptions) {
+      results.add(parseScheduleException(exception));
+    }
+    
+    return results;
+  }
+  
+  private ScheduleException parseScheduleException(ServiceCalendarDate exception) {
+    ScheduleException result = new ScheduleException();
+    result.setDate(parseServiceDateTime(exception.getDate()));
+    result.setType(parseScheduleExceptionType(exception.getExceptionType()));
+    return result;
+  }
+  
+  private String parseScheduleExceptionType(int type){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "parseScheduleExceptionType" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)